### DISCUSSION TITLE
A linter for Soufflé

### DISCUSSION THREAD

**Discussion by langston-barrett** [souffle-lang/souffle#2336] [Show and tell]
I wrote a linter for Soufflé! It's highly customizable; it's reasonably straightforward to define custom lints in the configuration file.

https://github.com/langston-barrett/souffle-lint

**Comment by b-scholz** [comment]
That is wonderful. Let's fork your branch under the souffle organisation to make your project more visible.

**Reply by langston-barrett** [reply]
@b-scholz You mean you'd like to move the repo from my account (i.e., `langston-barrett/souffle-lint`) to the Souffle organization (i.e., `souffle-lang/souffle-lint`)? I'd be happy to!

I think I already have enough permissions to make the switch. I'd like to retain enough access to be able to push commits and alter repository settings (e.g., setting up Github Pages, changing branch protection rules, etc.). If that sounds amenable to you, just comment here and I'll try to move it over!

> to make your project more visible

On this front, we could also consider adding it (and maybe `tree-sitter-souffle`) to the https://souffle-lang.github.io website. Perhaps on the "Applications" page, or even a new "Related Tools" page?

**Reply by b-scholz** [reply]
I have forked both repos under souffle-lang and made you admin.

**Comment by b-scholz** [comment]
Can you add both projects to the application list?
